__DEFINES/setup.dm

```diff
@@ -873,6 +873,7 @@ SEE_PIXELS	256
 #define ROLE_WIZARD     	"wizard"
 #define ROLE_GRINCH			"Grinch"
 #define ROLE_WEEABOO		"crazed weeaboo"
+#define ROLE_MADNESS		"something strange" //Time agents, chaos constructs, the zone menders, etc.
```

Comment:
might be better to roll it into minor roles, since the point was to stop preference creep